app-nordigen/tests/app-nordigen.e2e-spec.js

```diff
@@ -0,0 +1,191 @@
+import request from 'supertest';
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏great that you're taking initiative to write e2e test here, but I'd prefer we keep things simple in actual-server for now. Which means: stick to unit & integration tests instead. Would that be OK with you?

Reply:
Agreed, I don’t want to tie too much to the specific details of the frontend UI (although I would be more fine if the tests focused on clicking elements based on their label rather than CSS selectors). End to end testing is definitely something we need to figure out, though!